tensorrt_llm/_mnnvl_utils.py

```
@@ -338,6 +341,215 @@ def supports_mnnvl() -> bool:
 
+class HelixCpMnnvlMemory:
+    """MNNVL memory management for Helix context parallel (CP) operations."""
+
+    current_mem_offset: int = 0
+    current_rank_stride: int = 0
+    current_start_address: int = 0
+
+    # MPI communicator for CP
+    comm = None
+
+    allocated_map = {}
```

Ruff (0.14.8)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)